runtime/frequency/src/lib.rs

```diff
@@ -989,18 +989,8 @@ impl_runtime_apis! {
 			impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 			use orml_benchmarking::{add_benchmark as orml_add_benchmark};
 
-			let whitelist: Vec<TrackedStorageKey> = vec![
```

Comment:
Automatically getting all the whitelisted storages instead of hardcoding. Based on https://github.com/paritytech/substrate/pull/12205/files